### DISCUSSION TITLE
[Compatibility] The prettier and eslint config seem broken when use Yarn pnp

### DISCUSSION THREAD

**Discussion by nhattran998** [yarnpkg/berry#2990] [Q&A]
After I install SDK with command `yarn dlx @yarnpkg/pnpify --sdk vscode` + yarn 2.4.2 --> The prettier and eslint config is broken!

I already choose the version of Typescript of the project it's `4.3.2-pnpify` --> Its work.

Here it's some key in the log

<details>
<summary>Details</summary>

```
To use ESLint please install eslint by running yarn add eslint in the workspace folder alluvial
or globally using 'yarn global add eslint'. You need to reopen the workspace after installing eslint.

Alternatively, you can disable ESLint for the workspace folder alluvial by executing the 'Disable ESLint' command.
Failed to load the ESLint library for the document 
/.../projecname/.yarn/sdks/eslint/lib/api.js

Failed to load the ESLint library for the document 
/.../projecname/.yarn/sdks/eslint/bin/eslint.js

Failed to load the ESLint library for the document 
/.../projecname/.eslintrc.js
```

</details>

So please, anyone can show me, how I can integrate Eslint and Prettier with my config in the root folder(`.eslintrc.js` and `.prettierrc.js`)

**Comment by LaurensUP** [comment]
Did you figure anything out yet?
